docs/reference/services/project.rst

```diff
@@ -0,0 +1,96 @@
+.. py:currentmodule:: craft_application.services.project
+
+``ProjectService``
+==================
+
+The ``ProjectService`` is a service for handling access to this run's project.
+
+Project loading
+---------------
+
+The project service is responsible for loading, validating and rendering the project
+file to a ``Project`` model. While the service can render a project as though it is
```

Comment:
can you reference Project even if it is the base model?

Reply:
I've changed it, though the rendered version won't change because we don't include the class's reference in docs. I've created https://github.com/canonical/craft-application/issues/662 for that.